packages/main/src/RPA/Robocorp/Vault.py

```diff
@@ -107,20 +108,47 @@ class FileSecrets(BaseSecretManager):
           "key1": "value1"
         }
       }
+
+    OR
+
+    .. code-block:: YAML
+
+      name1:
+        key1: value1
+        key2: value2
+      name2:
+        key1: value1
     """
 
+    SERIALIZERS = {
+        ".json": (json.load, json.dump),
+        ".yaml": (yaml.full_load, yaml.dump),
```

Comment:
Read [here](https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation#how-to-disable-the-warning) about why I used `.full_load()` (instead of `.load()`).